src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -70,6 +70,8 @@ public void setTestVerbosity(){
 
     public static final String b37_2bit_reference_20_21 = largeFileTestDir + "human_g1k_v37.20.21.2bit";
 
+    public static final String b37_reference_20_21_img = largeFileTestDir + "human_g1k_v37.20.21.fasta.img";
```

Comment:
This should be moved to `GATKBaseTest`, because `BaseTest` should be clean from any repo-specific test resource...

Reply:
Fixed - thanks for letting me know